lib/matplotlib/backends/backend_tkagg.py

```diff
@@ -305,780 +18,17 @@ def blit(self, bbox=None):
-
-    def _get_toolbar(self):
-        if matplotlib.rcParams['toolbar'] == 'toolbar2':
-            toolbar = NavigationToolbar2TkAgg(self.canvas, self.window)
-        elif matplotlib.rcParams['toolbar'] == 'toolmanager':
-            toolbar = ToolbarTk(self.toolmanager, self.window)
-        else:
-            toolbar = None
-        return toolbar
-
-    def _get_toolmanager(self):
-        if rcParams['toolbar'] == 'toolmanager':
-            toolmanager = ToolManager(self.canvas.figure)
-        else:
-            toolmanager = None
-        return toolmanager
-
-    def resize(self, width, height=None):
-        # before 09-12-22, the resize method takes a single *event*
-        # parameter. On the other hand, the resize method of other
-        # FigureManager class takes *width* and *height* parameter,
-        # which is used to change the size of the window. For the
-        # Figure.set_size_inches with forward=True work with Tk
-        # backend, I changed the function signature but tried to keep
-        # it backward compatible. -JJL
-
-        # when a single parameter is given, consider it as a event
-        if height is None:
-            cbook.warn_deprecated("2.2", "FigureManagerTkAgg.resize now takes "
-                                  "width and height as separate arguments")
-            width = width.width
-        else:
-            self.canvas._tkcanvas.master.geometry("%dx%d" % (width, height))
-
-        if self.toolbar is not None:
-            self.toolbar.configure(width=width)
-
-    def show(self):
-        """
-        this function doesn't segfault but causes the
-        PyEval_RestoreThread: NULL state bug on win32
-        """
-        _focus = windowing.FocusManager()
-        if not self._shown:
-            def destroy(*args):
-                self.window = None
-                Gcf.destroy(self._num)
-            self.canvas._tkcanvas.bind("<Destroy>", destroy)
-            self.window.deiconify()
-        else:
-            self.canvas.draw_idle()
-        # Raise the new window.
-        self.canvas.manager.window.attributes('-topmost', 1)
-        self.canvas.manager.window.attributes('-topmost', 0)
-        self._shown = True
-
-    def destroy(self, *args):
-        if self.window is not None:
-            #self.toolbar.destroy()
-            if self.canvas._idle_callback:
-                self.canvas._tkcanvas.after_cancel(self.canvas._idle_callback)
-            self.window.destroy()
-        if Gcf.get_num_fig_managers()==0:
-            if self.window is not None:
-                self.window.quit()
-        self.window = None
-
-    def get_window_title(self):
-        return self.window.wm_title()
-
-    def set_window_title(self, title):
-        self.window.wm_title(title)
-
-    def full_screen_toggle(self):
-        is_fullscreen = bool(self.window.attributes('-fullscreen'))
-        self.window.attributes('-fullscreen', not is_fullscreen)
-
 
 @cbook.deprecated("2.2")
```

Comment:
Was this replaced by something in particular?

Reply:
This class was used by the **old** navigationtoolbar (pre-navigationtoolbar2) so basically unused for a very long time. But I moved it to _backend_tk like everyone else for a normal deprecation period.